Server/PrivateHeaders/XCTest/XCUIElementQuery.h

```diff
@@ -187,6 +186,11 @@
 - (id)sorted:(CDUnknownBlockType)arg1;
 - (id)uniqueMatchingSnapshotWithError:(id *)arg1;
 
+// Deprecated since Xcode 11.0
+- (XCElementSnapshot *)elementSnapshotForDebugDescription;
+// Added since Xcode 11.0
+- (XCElementSnapshot *)elementSnapshotForDebugDescriptionWithNoMatchesMessage:(id *)arg1;
+
```

Comment:
This file is overwritten by the script that does the class dump.

Move these methods to XCTest+CBXAdditions.h.

In the XCTest+CBXAdditions.m, use this to suppress warnings (also add the comments).

```
// This implementation does not implement all the methods in the category
	// interface.  This is by design - the category is used to expose private
// methods.  It is safe to ignore these warnings.
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wincomplete-implementation"
@implementation XCUIElementQuery (CBXAdditions)
 
- (XCElementSnapshot *)cbx_elementSnapshotForDebugDescription {

...

@end

#pragma clang diagnostic pop
```